src/app/[locale]/homepage/components/Support/Support.tsx

```diff
@@ -41,6 +42,7 @@ export default function Support() {
           />
           <Button
             variant="outlined"
```

Comment:
Double check how use button/links, i think we're very inconsistent with it.. 

It can be better to use:
```
import { Link } from "@/i18n/routing";
```
or
```
import { Link } from "@mui/material";
```
So the pages load faster/pre-compile... though shouldnt really be an issue with these fluff pages

Reply:
I don't think it's possible to use `Link` while also presenting the "outlined" button shape - `<Link variant="outlined">` doesn't work, so I think this is a legitimate use of this method in this case